project_task_category/__manifest__.py

```diff
@@ -8,6 +8,11 @@
     'version': '10.0.1.1.0',
     'author': 'Elico Corp, Odoo Community Association (OCA)',
     'license': 'AGPL-3',
+    'external_dependencies': {
```

Comment:
This shouldn't be added. It's not required for installing the module. Only for upgrading it.

Reply:
This pull request code is for upgrade the module.

Reply:
done